pkgs/by-name/be/bend/package.nix

```diff
@@ -0,0 +1,41 @@
+{ rustPlatform, fetchCrate, fetchFromGitHub, lib, stdenv }:
+
+let
+  hvm = rustPlatform.buildRustPackage rec {
+    pname = "hvm";
+    version = "2.0.12";
+    src = fetchCrate {
+      inherit pname version;
+      hash = "sha256-/55SK/5zBKXmucRQPoYt/8IHxisQlOxNEVMAZVMtCNI=";
+    };
+    RUSTC_BOOTSTRAP = true;
+    cargoHash = "sha256-9U8Y0KaQHIfOZnCKbl94VvjS/7Qmi6UnKMDZDTXcye0=";
+  };
+in
+rustPlatform.buildRustPackage rec {
+  pname = "Bend";
+  version = "0.2.9";
+
+  src = fetchFromGitHub {
+    owner = "HigherOrderCO";
+    repo = pname;
+    rev = "refs/tags/${version}";
+    hash = "sha256-MEfB2SBJN7uEwfZGoEL7DQXsr1fccdZyGyzHtNv9wow=";
+  };
+
+  cargoHash = "sha256-+i+Y3MgCBVN3REmPwAjm2SiF9FJ0i05czmPKB8JtAFM=";
+
+  RUSTC_BOOTSTRAP = true;
+
+  nativeBuildInputs = [ hvm ];
+
+  meta = with lib; {
+    description =
+      "Bend is a massively parallel, high-level programming language.";
+    homepage = "https://higherorderco.com/";
+    license = licenses.asl20;
+    mainProgram = "bend";
+    maintainers = with maintainers; [ k3yss ];
+    platforms = platforms.linux;
```

Comment:
Upstream does support darwin - is anything holding us back from having it supported here?

If it can't be worked around or fixed, please add a comment explaining the darwin blocker so that it is possible to address it in the future

Reply:
```suggestion
```

It works for me on aarch64-darwin.

Reply:
I have updated the code to use `platforms.unix` instead. Initially I was not considering the case where a user can have a external gpu for this/